README.md

```diff
@@ -2,7 +2,7 @@
 
 Gradle plugin that generates Java Documentation from an Android Gradle project.
 
-Works with the latest Gradle Android Tools version 3.0.1.
+Works with the latest Gradle Android Tools version 3.1.2.
```

Comment:
Can you revert? I'll update the README once I've published a new version

Reply:
Done